dsnp/index.ts

```diff
@@ -196,10 +196,11 @@ export const getSchema = (name: SchemaName): Deploy | null => {
 export type SchemaMapping = { [schemaName: string]: { [version: string]: number } };
 const chainMapping: { [genesisHash: string]: SchemaMapping } = {};
 
-export const GENESIS_HASH_TESTNET = "0x0c33dfffa907de5683ae21cc6b4af899b5c4de83f3794ed75b2dc74e1b088e72";
+export const GENESIS_HASH_TESTNET_ROCOCO = "0x0c33dfffa907de5683ae21cc6b4af899b5c4de83f3794ed75b2dc74e1b088e72";
+export const GENESIS_HASH_TESTNET_PASEO = "0x203c6838fc78ea3660a2f298a58d859519c72a5efdc0f194abd6f0d5ce1838e0";
 export const GENESIS_HASH_MAINNET = "0x4a587bf17a404e3572747add7aab7bbe56e805a5479c6c436f07f36fcc8d3ae1";
 
-chainMapping[GENESIS_HASH_TESTNET] = {
```

Comment:
Removing `GENESIS_HASH_TESTNET` from the exports is technically a breaking change to the public API, but it was a pretty recent addition and is not called out in the README. I did a quick search and see that I used it in the `oidc-provider-frequency` POC, but I can easily update that. On the other hand, if the intention is to remove `GENESIS_HASH_TESTNET_ROCOCO` at some point, maybe `GENESIS_HASH_TESTNET` should be retained as an alias for whichever the "preferred" testnet is at a point in time.

Reply:
> I used it in the `oidc-provider-frequency` POC

Actually it's redefined there rather than imported, so even less of an issue.